src/data/pages/tos.de.md

```diff
@@ -0,0 +1,229 @@
+
+Wir verwenden ausschliesslich sogenannte “Session-Cookies”. Diese werden bei einem Logout oder nach 12 Stunden automatisch gelöscht. Sie sind ausserdem zwingend notwendig für die Login-Funktion, du hast daher keine Möglichkeit dagegen Widerspruch zu erheben. Die Cookies werden aber auf deinem Rechner gespeichert. Du hast daher die volle Kontrolle und kannst diese vollständig löschen oder durch Änderung der Einstellungen in deinem Browser die Übertragung einschränken. Falls du Cookies für _eCamp_ deaktivierst, wirst du dich nicht mehr einloggen können und somit die Applikation nicht mehr nutzen können.
+
+#### Registrierung eines Nutzenden-Kontos
+
+Wir unterscheiden zwei Stufen von Nutzenden:
+
+1. Ohne Registrierung: Du kannst frei und anonym auf die rein informative Startseite [https://ecamp3.ch](/de/) zugreifen und dich auf ihr bewegen.
+2. Um die eigentlichen Lagerplanungs-Funktionen unserer Applikation zu nutzen, ist eine Registrierung erforderlich. Dazu muss ein Nutzenden-Konto erstellt werden mit einem zweiteiligen Namen, einer gültigen E-Mail-Adresse sowie einem Passwort. Die Registrierung beinhaltet einen Double-Opt-In, sprich während der Registrierung musst du zwingend die Nutzungsbedingungen und Datenschutzerklärung akzeptieren und du erhältst nach Ausfüllen des Registrierungsformulars zwecks Verifizierung eine Bestätigungs-E-Mail. Erst danach wird dein Nutzenden-Konto aktiviert. Von da an kannst du dein Nutzenden-Profil ergänzen oder auch das Passwort bei Bedarf ändern.
+
+#### Einsicht in deine Daten
+
+Sämtliche persönlichen Daten, die wir über dich dauerhaft speichern, kannst du jederzeit in deinem Nutzenden-Profil einsehen.
+
+#### Löschen deines Nutzenden-Kontos
+
+Möchtest du dein Konto nicht mehr weiterführen, so kannst du uns [kontaktieren](/de/kontakt) um eine Löschung zu beantragen. Dadurch werden alle von dir gespeicherten persönlichen Daten (nicht die Nutzendeninhalte) unwiderruflich gelöscht. Möchtest du _eCamp_ wieder nutzen, so musst du dich neu registrieren. Alle Nutzendeninhalte bleiben zwecks Transparenz, Nachvollziehbarkeit und Weiterverwendung in zukünftigen Lagern bestehen, werden jedoch einem anonymen Nutzenden-Konto zugewiesen.
+
+#### Kontaktaufnahme via Kontaktformular
+
+Du hast die Möglichkeit, das _eCamp Core Team_ via das [Kontaktformular](/de/kontakt) zu kontaktieren. Nimmst du mit uns Kontakt per E-Mail auf, werden folgende Daten verarbeitet:
+
+* E-Mail-Adresse
+* Inhalt deiner E-Mail
+* Betreff deiner E-Mail
+* Datum
+* Von dir angegebene Kontaktdaten (z.B. Name, Nachname)
+
+Deine Angaben werden zwecks Bearbeitung der Anfrage und für den Fall von Anschlussfragen bei uns gespeichert.
+
+Wir speichern deine Daten nur solange, wie dies zur Erledigung deiner Anfrage notwendig ist oder solange wir vom Gesetz dazu verpflichtet sind.
```

Comment:
d.h. die E-mails löschen wir dann wieder?

Reply:
Ja weiss halt auch nicht... :man_shrugging: das neue schweizer DSG weist uns explizit an, "die Bearbeitung der Personendaten auf das für den Verwendungszweck nötige Mindestmass beschränkt" zu halten (Art. 7 Abs. 3 [revDSG](https://www.fedlex.admin.ch/eli/fga/2020/1998/de)), wobei Speicherung explizit als eine Form der Bearbeitung genannt wird (Art. 5 Bst. d [revDSG](https://www.fedlex.admin.ch/eli/fga/2020/1998/de)).
Nach Abschluss einer Supportanfrage fällt es mir schwer zu argumentieren, dass wir für die Kontaktdaten weiterhin einen Verwendungszweck haben. Falls wir Learnings festhalten wollen müssten wir das also separat irgendwo anonymisiert machen.

Reply:
Ich finde, da könnten wir auch festlegen, dass wir die E-Mails nach einem Jahr löschen. Wir wollen ja keine Vorratsdatenspeicherung. Ich weiss aber auch nicht ob bei allen Anfragen klar ist wann eine Supportanfrage erledigt ist.

Reply:
> Ich finde, da könnten wir auch festlegen, dass wir die E-Mails nach einem Jahr löschen.

Müssen wir dann einfach auch tun. Oder die Frist dann jedes Jahr wieder um ein Jahr verlängern 😅

Reply:
## Core Meeting Decision

- Let's see what the legal commission of MSdS advises us